packages/tasks/src/hardware.ts

```diff
@@ -278,7 +278,7 @@ export const SKUS = {
 			},
 			"RTX 2080 Ti": {
 				tflops: 26.9,
-				memory: [11],
+				memory: [11, 22], //Modded 2080ti can be easily found in Taobao/Alibaba
```

Comment:
lol, I'm not sure we should enter this rabbit hole, there are plenty of unofficial mods out there. Let's see how it goes cc @Vaibhavs10